File: java/shop/src/main/java/de/epages/ws/update/UpdateServiceClient.java
package de.epages.ws.update;

import java.util.Calendar;

import de.epages.ws.update.stub.Port_Update;
import de.epages.ws.update.stub.TFindDeletes_Return;
import de.epages.ws.update.stub.TFindUpdates_Return;

public interface UpdateServiceClient extends Port_Update {

    /**
     * find deleted objects
     */
    public TFindDeletes_Return[] findDeletes(Calendar deletedAfter, String _class);

    /**
     * find updated objects
     */ public TFindUpdates_Return findUpdates(Calendar updatedAfter, String _class, String profile);
}

File: dotnet/shop/EpagesWebServices/UpdateServiceClient.cs
using System;
using EpagesWebServices.UpdateServiceRef;

namespace EpagesWebServices
{
    /// <summary>
    /// Client that uses generated stub to call UpdateService methods
    /// </summary>
    public class UpdateServiceClient : WebServiceConfiguration
    {
        UpdateService stub;

        public UpdateServiceClient()
        {
            stub = new UpdateService();
            stub.Credentials = new System.Net.NetworkCredential(WEBSERVICE_LOGIN, WEBSERVICE_PASSWORD);
            stub.PreAuthenticate = true;
            stub.Url = WEBSERVICE_URL;
        }

        public TFindUpdates_Return findUpdates(System.DateTime UpdatedAfter, string Class, string Profile)
        {
            return stub.findUpdates(UpdatedAfter, Class, Profile);
        }

        public TFindDeletes_Return findDeletes(System.DateTime DeletedAfter, string Class)
        {
            return stub.findDeletes(DeletedAfter, Class);
        }
    }
}